app/controllers/posts_controller.rb

```diff
@@ -32,7 +28,7 @@ def create
 
   def update
     @post = Post.find(params[:id])
-    if @post.user_id == 1 && (Time.now - @post.created_at < 10.minutes)
+    if @post.user_id == current_user.id && (Time.now - @post.created_at < 10.minutes)
```

Comment:
it might be nice to a method that tells you whether or not a post is more than 10 mins old. E.g. `@post.less_than_ten_minutes_old?`

What do you think?